pipecat-client-android-small-webrtc-transport/src/main/java/ai/pipecat/client/small_webrtc_transport/SmallWebRTCTransport.kt

```diff
@@ -174,6 +173,24 @@ class SmallWebRTCTransport internal constructor(
         }
     }
 
+    override fun deserializeConnectParams(
+        json: String,
+        startBotRequest: APIRequest
+    ): SmallWebRTCTransportConnectParams {
+        val startBotResult = JSON_INSTANCE.decodeFromString<SmallWebRTCStartBotResult>(json)
+
+        return SmallWebRTCTransportConnectParams(
```

Comment:
The `SmallWebRTCTransportConnectionParams` should also be able to receive the `iceConfig` from the `startBotResult`. So we can use it when connecting.